<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Calc3Window</class>
 <widget class="QWidget" name="Calc3Window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QScrollArea" name="scrollArea">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="widgetResizable">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="scrollAreaWidgetContents">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>270</width>
          <height>551</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_7">
         <item>
          <widget class="QGroupBox" name="displayGroup">
           <property name="title">
            <string>Отображать данные</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_5">
            <item>
             <widget class="QRadioButton" name="c1rbtn">
              <property name="text">
               <string>признака С1</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="c2rbtn">
              <property name="text">
               <string>признака С2</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="c3rbtn">
              <property name="text">
               <string>признака С3</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="calcrbtn">
              <property name="text">
               <string>расчета</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="srcrbtn">
              <property name="text">
               <string>исходные для коэф. преломления</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="rayParamGroup">
           <property name="title">
            <string>Расчет параметров излучения</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_6">
            <item>
             <widget class="QPushButton" name="calcParamPbtn">
              <property name="text">
               <string>Рассчитать признаки</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="statpbtn">
              <property name="text">
               <string>Рассчитать статистику</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="refractionGroup">
           <property name="title">
            <string>Коэффициент преломления</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <widget class="QGroupBox" name="loadGroup">
              <property name="title">
               <string>Загрузить данные</string>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_3">
               <item>
                <widget class="QRadioButton" name="loadfilerbtn">
                 <property name="text">
                  <string>Из файла</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QRadioButton" name="loadc1rbtn">
                 <property name="text">
                  <string>Из признака С1</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="loadpbtn">
                 <property name="text">
                  <string>Загрузить</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QGroupBox" name="calcGroup">
              <property name="title">
               <string>Расчет проводить</string>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_4">
               <item>
                <widget class="QRadioButton" name="dielrbtn">
                 <property name="text">
                  <string>для диэлектрической поверхности</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QRadioButton" name="metalrbtn">
                 <property name="text">
                  <string>для металлической поверхности</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="calcpbtn">
                 <property name="text">
                  <string>Рассчитать</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="currentIndex">
      <number>3</number>
     </property>
     <widget class="QWidget" name="pageC1">
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QTableView" name="C1table"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="pageC2">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QTableView" name="C2table"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="pageC3">
      <layout class="QVBoxLayout" name="verticalLayout_8">
       <item>
        <widget class="QTableView" name="C3table"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="pageCalc">
      <layout class="QVBoxLayout" name="verticalLayout_9">
       <item>
        <widget class="QTableView" name="statsTable">
         <property name="font">
          <font>
           <pointsize>11</pointsize>
          </font>
         </property>
         <attribute name="horizontalHeaderDefaultSectionSize">
          <number>80</number>
         </attribute>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
